pyaedt/modules/solutions.py

```diff
@@ -2418,81 +2378,219 @@ def _rotation_to_euler_angles(R):
         return np.array([x, y, z])
 
 
+class FfdSolutionDataExporter(FfdSolutionData):
+    """Export far field solution data.
+
+    Parameters
+    ----------
+    app : :class:`pyaedt.Hfss`
+        HFSS application instance.
+    sphere_name : str
+        Infinite sphere to use.
+    setup_name : str
+        Name of the setup. Make sure to build a setup string in the form of ``"SetupName : SetupSweep"``
+    frequencies : list
+        Frequency list to export. String with units is valid and float in Hertz.
```

Comment:
Does the list consist of a string specifying the units and then a float value in Hertz units? I think this description is unclear.